dbms/src/main/java/org/polypheny/db/partition/RangePartitionManager.java

```diff
@@ -174,24 +158,24 @@ public boolean validatePartitionSetup( List<List<String>> partitionQualifierList
                 lowerBound = temp;
 
                 // Rearrange List values lower < upper
-                partitionQualifierList.set( i, Stream.of( partitionQualifierList.get( i ).get( 1 ), partitionQualifierList.get( i ).get( 0 ) ).collect( Collectors.toList() ) );
+                partitionGroupQualifiers.set( i, Stream.of( partitionGroupQualifiers.get( i ).get( 1 ), partitionGroupQualifiers.get( i ).get( 0 ) ).collect( Collectors.toList() ) );
 
             } else if ( upperBound == lowerBound ) {
                 throw new RuntimeException( "No Range specified. Lower and upper bound are equal:" + lowerBound + " = " + upperBound );
             }
 
-            for ( int k = i; k < partitionQualifierList.size() - 1; k++ ) {
-                int contestingLowerBound = Integer.parseInt( partitionQualifierList.get( k + 1 ).get( 0 ) );
-                int contestingUpperBound = Integer.parseInt( partitionQualifierList.get( k + 1 ).get( 1 ) );
+            for ( int k = i; k < partitionGroupQualifiers.size() - 1; k++ ) {
+                int contestingLowerBound = Integer.parseInt( partitionGroupQualifiers.get( k + 1 ).get( 0 ) );
+                int contestingUpperBound = Integer.parseInt( partitionGroupQualifiers.get( k + 1 ).get( 1 ) );
 
                 if ( contestingUpperBound < contestingLowerBound ) {
                     int temp = contestingUpperBound;
                     contestingUpperBound = contestingUpperBound;
```

Comment:
This makes no sense...

Reply:
Thx, copy& paste while refactoring.
Fixed.

That is worth a separate test. I'll write a todo